### PR TITLE
feat: add `SPC t m` to toggle minimap

### PR DESCRIPTION
**Pull request by MarcoIeni**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7566,6 +7566,13 @@
                                         "type": "command",
                                         "command": "editor.action.toggleWordWrap"
                                     },
+                                    {
+                                        "key": "m",
+                                        "name": "Toggle minimap",
+                                        "icon": "list-selection",
```
Comment (stevenguh) [inline package.json:7572]: Would `symbol-ruler`, `preview`, `output`, `note`  be a better fit icon?

**Comment by stevenguh** [comment]
Although, this doesn't seem to exist in spacmacs (?), seems reasonable to me.

**Comment by MarcoIeni** [comment]
Actually, now that I think about it, it's more suitable for `T` (UI toggles). what do you think?

File: package.json (hunk 1 above)
Comment (MarcoIeni) [inline package.json:7572]:
```suggestion
                                        "icon": "symbol-ruler",
```

**Comment by stevenguh** [comment]
> Actually, now that I think about it, it's more suitable for `T` (UI toggles). what do you think?

Sounds reasonable 👍🏼

**Comment by stevenguh** [comment]
Thanks merged! Will get a release out this weekend.

**Pull request merged**